### PR TITLE
Fixing typos and punctuation

### LINKED ISSUE #2018: [Change Request]: Manage garrison options misspelled
### Is the requested change related to a problem?

The button for placing watchposts or roadblocks is wrongly named as "outposts or roadblocks".

Only problem really is just the wording is misleading or bugging me a small bit. Of course, the description is true. 

image

### Describe the solution you'd like.

Just a quick renaming to "Watchposts or roadblocks".

### Alternative solutions

Ignoring this little detail, not a biggie either way.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Bob-Murphy**
adapted wording

## What type of PR is this.
1. [x] Bug - not really, but wording
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
    - Fixed wording in STR_antistasi_dialogs_build_outpost_button_text
    - Changes "Assasination" to "Assassination" where available
    - few punctuation things
    - some grammar fixes

### Please specify which Issue this PR Resolves.
closes #2018 

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

**Review by doommetal — APPROVED** [review]

File: A3-Antistasi/REINF/dismissPlayerGroup.sqf [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+if (player != leader group player) exitWith {["Dismiss Group", "You cannot dismiss anyone if you are not the squad leader."] call A3A_fnc_customHint;};
 
 private ["_units","_hr","_resourcesFIA","_unit","_newGroup"];
 
 _units = _this select 0;
 _units = _units - [player];
 _units = _units select { !(isPlayer _x) && { !(_x == petros) } };
 if (_units isEqualTo []) exitWith {};
-if (_units findIf {!([_x] call A3A_fnc_canFight)} != -1) exitWith {["Dismiss Group", "You cannot disband supressed, undercover or unconscious units"] call A3A_fnc_customHint;};
+if (_units findIf {!([_x] call A3A_fnc_canFight)} != -1) exitWith {["Dismiss Group", "You cannot disband supressed, undercover or unconscious units."] call A3A_fnc_customHint;};
```
Comment (jaj22) [inline A3-Antistasi/REINF/dismissPlayerGroup.sqf:11]: supressed -> suppressed

File: A3-Antistasi/functions/AI/fn_artySupport.sqf [hunk 2]
```diff
@@ -59,11 +59,11 @@ if ((_veh != _soldierX) and (not(_veh in _artyArray))) then
 	};
 } forEach _unitsX;
 
-if (!_hasArtillery) exitWith {["Artillery Support", "You must select an artillery group or it is a Mobile Mortar and it's moving"] call A3A_fnc_customHint;};
-if (!_areAlive) exitWith {["Artillery Support", "All elements in this Batery cannot fire or are disabled"] call A3A_fnc_customHint;};
-if ((_hasAmmunition < 2) and (!_areReady)) exitWith {["Artillery Support", "The Battery has no ammo to fire. Reload it on HQ"] call A3A_fnc_customHint;};
-if (!_areReady) exitWith {["Artillery Support", "Selected Battery is busy right now"] call A3A_fnc_customHint;};
-if (_typeAmmunition == "not_supported") exitWith {["Artillery Support", "Your current modset doesent support this strike type"] call A3A_fnc_customHint;};
+if (!_hasArtillery) exitWith {["Artillery Support", "You must select an artillery group or it is a Mobile Mortar and it's moving."] call A3A_fnc_customHint;};
+if (!_areAlive) exitWith {["Artillery Support", "All elements in this Batery cannot fire or are disabled."] call A3A_fnc_customHint;};
```
Comment (jaj22) [inline A3-Antistasi/functions/AI/fn_artySupport.sqf:63]: Batery -> Battery

File: A3-Antistasi/functions/Dialogs/fn_clearForest.sqf [hunk 3]
```diff
@@ -1,4 +1,4 @@
-if (player != theBoss) exitWith {["Clean Forest", "Only Commanders can order to clean the forest"] call A3A_fnc_customHint;};
+if (player != theBoss) exitWith {["Clean Forest", "Only Commanders can order to clean the forest."] call A3A_fnc_customHint;};
 { [_x, true] remoteExec ["hideObjectGlobal",2] } forEach (nearestTerrainObjects [getMarkerPos respawnTeamPlayer,["tree","bush","small tree"],70]);
-["Clean Forest", "You've cleared the surroundings of trees and bushes"] call A3A_fnc_customHint;
+["Clean Forest", "You've cleared the surroundings of trees and bushes."] call A3A_fnc_customHint;
```
Comment (jaj22) [inline A3-Antistasi/functions/Dialogs/fn_clearForest.sqf:3]: "Clean" here is dead weird. Should be "Clear".

File: A3-Antistasi/functions/Dialogs/fn_fastTravelRadio.sqf [hunk 4]
```diff
@@ -131,15 +131,15 @@ if (count _positionTel > 0) then
 			//_unit hideObject false;
 		} forEach units _groupX;
 		//if (!_esHC) then {sleep _distanceX};
-		if (!_esHC) then {disableUserInput false;cutText ["You arrived to destination","BLACK IN",1]} else {["Fast Travel", format ["Group %1 arrived to destination",groupID _groupX]] call A3A_fnc_customHint;};
+		if (!_esHC) then {disableUserInput false;cutText ["You arrived to destination.","BLACK IN",1]} else {["Fast Travel", format ["Group %1 arrived to destination.",groupID _groupX]] call A3A_fnc_customHint;};
```
Comment (jaj22) [inline A3-Antistasi/functions/Dialogs/fn_fastTravelRadio.sqf:134]: "You arrived at the destination."

File: A3-Antistasi/functions/Dialogs/fn_fastTravelRadio.sqf [hunk 5]
```diff
@@ -131,15 +131,15 @@ if (count _positionTel > 0) then
...
 		if (_forcedX) then {forcedSpawn = forcedSpawn - [_base]};
 		[] call A3A_fnc_playerLeashRefresh;
 		sleep 5;
 		{_x allowDamage true} forEach units _groupX;
 		}
 	else
 		{
-		["Fast Travel", "You must click near marker under your control"] call A3A_fnc_customHint;
+		["Fast Travel", "You must click near marker under your control."] call A3A_fnc_customHint;
```
Comment (jaj22) [inline A3-Antistasi/functions/Dialogs/fn_fastTravelRadio.sqf:142]: near *a* marker

**Pull request merged**